metrics/report/README.md

```diff
@@ -0,0 +1,12 @@
+# PnP failure Report tool
```

Comment:
Comment for all the files.
Given the other tools in this repo live under the `cmd` directory, I would consider moving this to `cmd/reporttool` to match.
Sure, it is metrics specific, but I didn't place `checkmetrics` in the metrics dir - so we should probably follow suit.